spec/controllers/projects/contributions_controller_spec.rb

```diff
@@ -110,6 +124,54 @@
     end
   end
 
+  describe 'project' do
+    let(:user){ create(:user).decorate }
+
+    before do
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+    end
+
+    context 'when the project state is different of online' do
+      let(:project) { create(:project, :failed) }
+
+      it 'should redirect to root path on get new request' do
+        get :new, { locale: :pt, project_id: project.id }.merge(ssl_options)
+        is_expected.to redirect_to root_path
+      end
+
+      it 'should redirect to root path on post create request' do
+        post :create, { locale: :pt, project_id: project.id, contribution: attributes_for(:contribution) }.merge(ssl_options)
+        is_expected.to redirect_to root_path
+      end
+    end
+
+    context 'when the project is online' do
+      it_behaves_like "when no hooks are triggered on new and create methods" do
```

Comment:
You're using double quotes here `"` when in all the other specs you use single ones `'`. It'd be better to keep the pattern 💅